torch/distributed/algorithms/_checkpoint/checkpoint_wrapper.py

```diff
@@ -227,13 +227,13 @@ def apply_activation_checkpointing_wrapper(
         check_fn = lambda l: isinstance(l, nn.Linear)
         apply_activation_checkpointing(model, checkpoint_wrapper_fn=checkpoint_wrapper, check_fn=check_fn)
     Args:
-        module (nn.Module):
-            The model who's submodules (or self) should be wrapped with activation checkpointing.
+        model (nn.Module):
+            The model whose submodules should be wrapped with activation checkpointing.
         checkpoint_wrapper_fn (Optional[Callable[nn.Module]])
             A `Callable` which will wrap modules
```

Comment:
nit (before PR): 
```suggestion
            A ``Callable`` which will wrap modules
```
I think single backtick in docstrings are rendered as italicized, while double backtick is code type.